projects/demo-playwright/tests/kit/input-date-range/input-date-range.spec.ts

```diff
@@ -0,0 +1,115 @@
+import {expect, Locator, test} from '@playwright/test';
+
+import {TuiDocumentationPagePO, tuiGoto} from '../../../utils';
+import {TuiInputDateRangePO} from '../../../utils/page-objects/input-date-range.po';
+
+test.describe(`InputDateRange`, () => {
+    let example!: Locator;
+    let inputDateRange!: TuiInputDateRangePO;
+
+    test.use({
+        viewport: {width: 650, height: 500},
```

Comment:
Do we actually need the such wide screenshots ?

Reply:
I toke sizes from similar test with cypress